bundles/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/internal/handler/TeslaEventEndpoint.java

```diff
@@ -54,25 +54,20 @@ public class TeslaEventEndpoint implements WebSocketListener, WebSocketPingPongL
     private final Logger logger = LoggerFactory.getLogger(TeslaEventEndpoint.class);
 
     private String endpointId;
-    protected WebSocketFactory webSocketFactory;
 
     private WebSocketClient client;
     private ConnectionState connectionState = ConnectionState.CLOSED;
     private @Nullable Session session;
-    private EventHandler eventHandler;
+    private @Nullable EventHandler eventHandler;
     private final Gson gson = new Gson();
 
     public TeslaEventEndpoint(ThingUID uid, WebSocketFactory webSocketFactory) {
-        try {
-            this.endpointId = "TeslaEventEndpoint-" + uid.getAsString();
+        this.endpointId = "TeslaEventEndpoint-" + uid.getAsString();
 
-            String name = ThingWebClientUtil.buildWebClientConsumerName(uid, null);
-            client = webSocketFactory.createWebSocketClient(name);
-            this.client.setConnectTimeout(TIMEOUT_MILLISECONDS);
-            this.client.setMaxIdleTimeout(IDLE_TIMEOUT_MILLISECONDS);
-        } catch (Exception e) {
-            throw new RuntimeException(e);
-        }
```

Comment:
@kaikreuzer  without a debug log it is hard to find the culprit. These lines might be related. I would not expect a runtimeexception here so i removed it. Maybe i shouldn't have done that.

Any way a log would be highly appreciated.